troubleshooting/Ingestion/Readme.md

```diff
@@ -0,0 +1,13 @@
+# Troubleshooting Ingestion
+
+## Fiddler
+
+A tool such as Fiddler can be used to record the raw traffic between the SDK and Ingestion Service.
```

Comment:
We  could say something like 
A tool such as Fiddler can be used to inspect raw HTTPS data from an app integrated with SDK to Ingestion Service.